workflow-templates/im-build-dotnet-ci.yml

```diff
@@ -360,6 +353,25 @@ jobs:
           branch-name: ${{ github.head_ref }} # This is only populated when the trigger is pull_request, otherwise it is empty
           # tag-prefix: '' # TODO:  If you prefix your tags like (mfe/db/etc) uncomment and add that value here
 
+      # TODO:  If there are multiple deployables, duplicate this step once per deployable project and update the PROJECT_ROOT, PUBLISHED_APP and DEPLOY_ZIP variables with the updated names.
+      - name: Build and Publish
+        working-directory: ${{ env.PROJECT_ROOT }}
+        run: |
+          VERSION_ONLY=${NEXT_VERSION#?}; # Removes a leading 'v'. Use a different expression to strip a tag-prefix if you've specified one.
+          dotnet build --configuration Release /p:Version=$VERSION_ONLY
+          dotnet publish --configuration Release --no-build -o ./${{ env.PUBLISHED_APP }}
+          (cd ${{ env.PUBLISHED_APP }} && zip -r ../${{ env.DEPLOY_ZIP }} .)
+
+      # TODO:  On a Windows build runner, the 'zip' commandline tool isn't available. Use PowerShell instead
+      # - name: Build and Publish
+      #   working-directory: ${{ env.PROJECT_ROOT }}
+      #   shell: pwsh
+      #   run: |
+      #     $VERSION_ONLY = ($env:NEXT_VERSION).substring(1) # Removes a leading 'v'. Use a different expression to strip a tag-prefix if you've specified one.
+      #     dotnet build --configuration Release /p:Version=$VERSION_ONLY
+      #     dotnet publish --configuration Release --no-build -o ./${{ env.PUBLISHED_APP }}
+      #     Compress-Archive -Path ./${{ env.PUBLISHED_APP }}/* -DestinationPath ${{ env.DEPLOY_ZIP }}
```

Comment:
Should we do this all with powershell instead of having two different steps?

Reply:
Yeah, probably.